Modules/_collectionsmodule.c

```diff
@@ -1508,15 +1508,13 @@ deque_init(dequeobject *deque, PyObject *args, PyObject *kwdargs)
 static PyObject *
 deque_sizeof(dequeobject *deque, void *unused)
 {
-    Py_ssize_t res;
-    Py_ssize_t blocks;
-
-    res = _PyObject_SIZE(Py_TYPE(deque));
+    size_t res = _PyObject_SIZE(Py_TYPE(deque));
```

Comment:
Doesn't it need an explicit cast to size_t?

Reply:
I chose to limit this change only to *prepare* the code for _PyObject_SIZE() API change.

With commit 131801d14dfc4f0b2b79103612c88e2e282ff158, _PyObject_SIZE() return type is now well defined: it returns a ``size_t``.